### ISSUE TITLE
Docker Container is reachable even when the Host Machine / Local VM is powered off !!!!

### ISSUE THREAD

**Issue by Arun129** [damsonn/hue-state#3]
Hi,

I have installed docker in my local vm and created a tomcat container. In that tomcat containers it has been deployed with the sample war file.

Here the problem is, I have powered off my local vm but Still am able to access the deployed webpage of the tomcat container.

Things I have done :

1) Tried in Chrome and Firefox even after and before clearing the browser cache and cookies.
2) Tried in Icognito Mode in chrome and firefox

**Comment by Arun129** [comment]
Hi Guys,

This is very fishy..still unresolvable. 

You feedback or comments will be very helpful to proceed further..

**Comment by damsonn** [comment]
Hi @Arun129 

I am not sure why you are using Tomcat at all... Could you elaborate ?

But regarding your docker issue, it seems like it's a docker issue than anything to do with this project.

Cheers,
Damien

**Comment by Arun129** [comment]
Hi @damsonn 

Actually I have just deployed a sample war in that tomcat container. That container is just running in my local docker host. If I have stopped that container / If I am shutting down that docker host machine.

Still am able to browse the web page of sample war file which I have deployed. Even If am ping that local docker host IP - it is showing destination host unreacheable, but that deployed war page is accessable... 

Thanks
Arun
